### ISSUE TITLE
App Crashed

### ISSUE THREAD

**Issue by ask9027** [MuntashirAkon/AppManager#1228]
### Please check before submitting an issue

- [X] I know what my device, OS and App Manager versions are
- [X] I know how to take logs
- [X] I know how to reproduce the issue which may not be specific to my device

### Describe the bug

App crashed when come back to home page.

### To Reproduce

Click on Search Bar  then keyboard will open(dont type anything),
Now click on any item of recycler view, then it will next to item related activity, but app crashed while come back to previews pages(Home Page).
here is short video.

https://github.com/MuntashirAkon/AppManager/assets/61283438/51ab87dc-3342-4048-801e-76839ded7d27

 

### Logs

```bash
java.lang.RuntimeException: java.lang.IllegalArgumentException: ApkFile not found for key -795078739
    at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel.lambda$loadAppInfo$1(AppInfoViewModel.java:330)
    at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel$$ExternalSyntheticLambda1.run(Unknown Source:2)
    at android.os.Handler.handleCallback(Handler.java:942)
    at android.os.Handler.dispatchMessage(Handler.java:99)
    at android.os.Looper.loopOnce(Looper.java:211)
    at android.os.Looper.loop(Looper.java:300)
    at android.app.ActivityThread.main(ActivityThread.java:8296)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:559)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:954)
 Caused by: java.lang.IllegalArgumentException: ApkFile not found for key -795078739
   at io.github.muntashirakon.AppManager.apk.ApkFile.getInstance(ApkFile.java:105)
   at io.github.muntashirakon.AppManager.details.info.AppInfoViewModel.loadAppInfo(AppInfoViewModel.java:261)
   at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.loadPackageInfo(AppInfoFragment.java:1617)
   at io.github.muntashirakon.AppManager.details.info.AppInfoFragment.$r8$lambda$oxlKbqiAsxIkEG69gntC-jrgnXk(Unknown Source:0)
   at io.github.muntashirakon.AppManager.details.info.AppInfoFragment$$ExternalSyntheticLambda63.run(Unknown Source:2)
   at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:487)
   at java.util.concurrent.FutureTask.run(FutureTask.java:264)
   at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
   at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
   at java.lang.Thread.run(Thread.java:1012)
```

Device Info:
App version: 3.1.3
App version code: 426
Android build version: V14.0.2.0.TKQINXM
Android release version: 13
Android SDK version: 33
Android build ID: TKQ1.221013.002 test-keys
Device brand: Xiaomi
Device manufacturer: Xiaomi
Device name: courbetin
Device model: M2101K9AI
Device product name: courbetin
Device hardware name: qcom
ABIs: [arm64-v8a, armeabi-v7a, armeabi]
ABIs (32bit): [armeabi-v7a, armeabi]
ABIs (64bit): [arm64-v8a]
System language: en-IN
In-App Language: auto
Mode: auto
Inferred Mode: root

**Comment by WikiBayer** [comment]
+ 1 crash on Android 14

**Comment by MuntashirAkon** [comment]
Fixed in a10b4fb90bd83212f09286d6aaffc7090ebdb354
